.github/workflows/p-diff-sync-tests.yml

```
@@ -11,364 +10,49 @@ on:
-  #     uses: denoland/setup-deno@v1
-  #     with:
-  #       deno-version: v1.32.4
-
-  #   - run: rustup target add wasm32-unknown-unknown
-  #   - run: pnpm install
-  #   - run: cargo install holochain_cli --version 0.2.2 || echo "hc already installed"
-  #   - run: pnpm run build
-  #   - run: cd bootstrap-languages/p-diff-sync && pnpm run integration-test
\ No newline at end of file
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)